staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch.go

```diff
@@ -866,7 +871,11 @@ func StrategicMergeMapPatch(original, patch JSONMap, dataStruct interface{}) (JS
 	// each field in a strategic merge patch, we can't find the go struct tags. Hence, we can't easily  do a strategic merge
 	// for custom resources. So we should fail fast and return an error.
 	if _, ok := dataStruct.(*unstructured.Unstructured); ok {
-		return nil, mergepatch.ErrUnsupportedStrategicMergePatchFormat
+		schema, err = NewPatchMetaFromStruct(MetaOnly{})
```

Comment:
What does this mean for patches again fields outside of meta? Will they apply like to a native type without any tags?

Reply:
If yes, I would like to see a test case. If not, it must be rejected.

Reply:
only supporting a patch type on part of an object is bizarre to me